src/main/scala/com/codacy/clangtidy/ClangTidyReportParser.scala

```diff
@@ -0,0 +1,19 @@
+package com.codacy.clangtidy
+
+import java.nio.file.Paths
+
+class ClangTidyReportParser {
+
+  private val ResultRegex =
+    "(.+|[a-zA-Z]:\\\\.+):([0-9]+):([0-9]+): ([^:]+): (.+) \\[(.+?)\\]".r
+
+  def parse(lines: Seq[String]): Seq[ClangTidyResult] = {
+    lines.collect {
+      case ResultRegex(pathStr, line, column, level, txt, checksList) =>
+        val path = Paths.get(pathStr)
+        val check = checksList.split(",").head
```

Comment:
In case of exception better give an error message instead of "head of empty list"

Reply:
Fixed